oak_crypto/src/encryptor.rs

```diff
@@ -204,10 +204,18 @@ impl ClientEncryptor {
             .encrypted_message
             .as_ref()
             .context("response doesn't contain encrypted message")?;
+        let nonce = encrypted_message.nonce.clone().try_into().map_err(|_| {
```

Comment:
Could you annotate `nonce` with the expected type (even though it is not necessary, it is good to see it explicitly). In fact, maybe define a helper function to convert the nonce and deal with the error, since it's already implemented in two places.

Reply:
Done